### ISSUE TITLE
Private repository / hosting

### ISSUE THREAD

**Issue by logan-anderson** [tinacms/tina-starter-alpaca#33]
I made a branch a while ago called private-repo to test whether or not I could get this project working from a private repo. At the time it did not work but it turns out my PR to tinaCMS to fix #32 also fixed the private repo issues I was having. There are still a couple of issues that remain.

# Who generates the github oAuth keys?
When testing I generated the keys. Using my keys worked but its kinda awkward for the user because they had to request access to use the tinacms/staff team repos. As far as I can tell, one can only do this the very first time when they authenticate with github and as a further annoyance they will have to wait until an admin from tinacms/staff team can verify them.

How can we fix this?

**I think** if tinacms/staff team generates the keys then everyone who is staff should have access. not sure exactly how this would be done

# hosting

I think it would be best if an admin from tinaCMS/staff team hosted under the tinaCMS vercel account. (maybe that would also solve the above problem)

I could host it but then I would have to request access to from to tinaCMS/staff for vercel use. We would also have the issue with the oAuth key above. 
 
# PR branches for hosting

When we eventually host on vercel we will get a hosted version of the site for PRs. The problem with this is that the oAuth keys will not work as they are tied to a specific URL. (the PR urls are effectively random). My suggestion would be that we test PR locally (on localhost) and have a hosted staging or dev environments on vervel that we can test from. Once those tested there then we can merge into master.  

# In short
* I'd love to host it but it seems that there are to many drawbacks
* To overcome these drawbacks an admin from tinacms/staff could generate the keys and host it on the tinaCMS vervel account
* PR branches for testing open authoring will not work easily
